og_migrate/migrations/d7_og_user_membership.yml

```diff
@@ -0,0 +1,55 @@
+id: d7_og_user_membership
+label: Group user memberships
+migration_tags:
+  - Drupal 7
+source:
+  plugin: d7_og_membership
+  entity_type: user
+process:
+  id: id
+  type:
+    -
+      plugin: migration_lookup
+      migration: d7_og_membership_type
+      source: type
+    -
+      plugin: skip_on_empty
+      method: row
+      message: 'Membership type is missing'
+  uid:
+    -
+      plugin: migration_lookup
+      migration: d7_user
+      source: etid
+    -
+      plugin: skip_on_empty
+      method: row
+      message: 'User uid is missing'
+  entity_id: gid
+  entity_type:
+    -
+      plugin: og_entity_type_exists
+      source: group_type
+      bundle_property: entity_bundle
+    -
+      plugin: skip_on_empty
+      method: row
+      message: 'Entity type is missing'
+  roles:
+    -
+      plugin: migration_lookup
+      migration: d7_og_role
+      source: roles
+  state: state
```

Comment:
State column in Drupal 7 is a varchar but it contains an integer (1, 0).

```suggestion
  state:
    -
      plugin: static_map
      source: state
      map:
        0: pending
        1: active
      default_value: 1
```

Reply:
FYI, you can click on commit suggestion:

<img width="877" alt="commit github suggestion" src="https://user-images.githubusercontent.com/1823998/54191241-d41dc180-44b5-11e9-8d4e-ca6889ca0d38.png">